docs/master/api-reference/directives.md

```diff
@@ -1756,6 +1757,16 @@ type Mutation {
 }
 ```
 
+If you have an array you need to inject a value into, you can use an asterisk `*`. 
+
+```graphql
+type Mutation {
+  createTask(input: CreateTaskInput!): Task
+    @create
+    @inject(context: "user.id", name: "input.*.user_id")
```

Comment:
This example does not work.

Reply:
I dont know if the docs has an existing example we can use, but I've updated it to a `updateUser` mutation.